packages/terra-application/src/terra-dev-site/app/components.2/ApplicationNavigation.app.mdx

```diff
@@ -1,4 +1,4 @@
-import PropsTable from '@cerner/terra-application/lib/application-navigation/ApplicationNavigation?dev-site-props-table';
+// import PropsTable from '@cerner/terra-application/lib/application-navigation/ApplicationNavigation?dev-site-props-table';
```

Comment:
this is outdated doc. I commented out the props table to prevent a dev site error. this will be cleaned up in a future PR